website/blog/2023-05-04-generating-dynamic-docs.md

```diff
@@ -0,0 +1,368 @@
+---
+title: "Generating dynamic dbt docs for a whole domain at once"
```

Comment:
Not sure about this title - dynamic to me implies that they're self-updating over time. This programmatically finds models and columns to document, but once they're documented there's no more dynamism.